lib/types/api.types.ts

```diff
@@ -271,56 +292,70 @@ export type DialogAppSDK = Omit<BaseAppSDK, 'ids'> & {
   window: WindowAPI
 }
 
-export type PageAppSDK = Omit<BaseAppSDK, 'ids'> & {
+export type PageAppSDK<
+  InstallationParameters extends KeyValueMap = KeyValueMap,
+  InstanceParameters extends KeyValueMap = KeyValueMap,
+  InvocationParameters extends SerializedJSONValue = { path: string }
+> = Omit<BaseAppSDK<InstallationParameters, InstanceParameters, InvocationParameters>, 'ids'> & {
   /** A set of IDs actual for the app */
   ids: Omit<IdsAPI, EntryScopedIds>
 }
 
-export type HomeAppSDK = Omit<BaseAppSDK, 'ids'> & {
+export type HomeAppSDK<
+  InstallationParameters extends KeyValueMap = KeyValueMap,
+  InstanceParameters extends KeyValueMap = KeyValueMap
+> = Omit<BaseAppSDK<InstallationParameters, InstanceParameters, never>, 'ids'> & {
   ids: Omit<IdsAPI, EntryScopedIds>
 }
 
-export type ConfigAppSDK = Omit<BaseAppSDK, 'ids'> & {
+export type ConfigAppSDK<
+  InstallationParameters extends KeyValueMap = KeyValueMap,
+  InstanceParameters extends KeyValueMap = KeyValueMap
+> = Omit<BaseAppSDK<InstallationParameters, InstanceParameters, never>, 'ids'> & {
```

Comment:
Config location has no instance params

```suggestion
export type ConfigAppSDK<InstallationParameters extends KeyValueMap = KeyValueMap> = Omit<
  BaseAppSDK<InstallationParameters, never, never>,
  'ids'
> & {
```